dev/conf/default.conf

```diff
@@ -0,0 +1,20 @@
+server {
+    listen              443 ssl;
+    server_name         proxy;
+    access_log          /var/log/nginx/access.log;
+
+    ssl_certificate     /etc/nginx/tls/nginx.crt;
+    ssl_certificate_key /etc/nginx/tls/nginx.key;
+    ssl_verify_client   optional_no_ca;
+    ssl_verify_depth    10; # for good measure
```

Comment:
what do you mean by "for good measure"? can you elaborate please?

Reply:
I copied this configuration from another repo, together with this comment...
I will remove this line, it's not mandatory anyway.

Reply:
done